docs/api/structures/shortcut-details.md

```diff
@@ -0,0 +1,15 @@
+# ShortcutDetails
+
+* `target` String - The target to launch from this shortcut.
+* `cwd` String (optional) - The working directory. Default is empty.
+* `args` String (optional) - The arguments to be applied to `target` when
+launching from this shortcut. Default is empty.
```

Comment:
Worth indenting these 2 spaces? The other new files seem to usually indent the second line of list items.